spring-cloud-starter-stream-source-websocket/src/main/java/org/springframework/cloud/stream/app/websocket/source/WebsocketSourceProperties.java

```diff
@@ -0,0 +1,62 @@
+/*
+ * Copyright 2014-2016 the original author or authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ */
+
+package org.springframework.cloud.stream.app.websocket.source;
+
+import org.springframework.boot.context.properties.ConfigurationProperties;
+
+/**
+ * Command line arguments available for {@link WebsocketSourceConfiguration}
+ *
+ * @author Krishnaprasad A S
+ *
+ */
+@ConfigurationProperties("websocket")
+public class WebsocketSourceProperties {
+
+	public static final String DEFAULT_PATH = "/websocket";
+
+	private static final String DEFAULT_ALLOWED_ORIGINS = "*";
+
+	/**
+	 * the path on which a WebsocketSource consumer needs to connect. Default is
```

Comment:
Why does the sentence start not from capital letter ?
  
And somehow the sentence is misleading:
> The path on which server WebSocket handler is exposed